delayed_assert/test_delayed_assert.py

```diff
@@ -1,9 +1,9 @@
 # from ..delayed_assert import expect, assert_expectations
-
+import unittest
 from delayed_assert import expect, assert_expectations
 
 
-class TestDelayedAssert:
+class TestDelayedAssert(unittest.TestCase):
```

Comment:
When writing test I had py.test in mind. This can be added as the requirement or in Doc. But I would certainly don't want ```unittest```(just a personal thing but open to suggestions)